docs/kubernetes.md

```diff
@@ -35,12 +35,26 @@ cluster that is needed for other purposes.
 [Cloud Foundry Acceptance Tests]: https://github.com/cloudfoundry/cf-acceptance-tests
 
 ### StatefulSet
-Fissile emits a [StatefulSet] under two circumstances.  Any self-clustering
-roles (i.e. any role with the `clustered` tag) will be a StatefulSet, in order
-for each pod to be addressable (so that they can talk to each other).  For
-example, a NATS role would fall under this category.  Secondly, any roles which
-require local storage will be a StatefulSet to take advantage of volume claim
-templates.
+
+Fissile emits a [StatefulSet] under two circumstances.
+
+Any self-clustering roles (i.e. any role with the `clustered` tag)
+will be a StatefulSet, in order for each pod to be addressable (so
+that they can talk to each other). For example, a NATS role would fall
+under this category.
```

Comment:
Except that [NATS is indexed](https://github.com/SUSE/scf/blob/b5e5fd51eccb736bbf8dd9f7d92e802eefc4df7e/container-host-files/etc/scf/config/role-manifest.yml#L101). (Probably for the service in front of it doing LB work)

Reply:
Ok, so NATs changed at some point and nobody bothered to update the fissile docs (most likely not remembered that it was referenced in them). So, ok, I can do that, i.e. look for a new example of a clustered role. (NATS has the big example either in fise or configuration.md).

Reply:
Done with commit 345848333b28aa12daf75028b9ad103163fe782c.
Kept NATS, rewrote to `indexed`, then went with `mysql` for a clustered role.